store/todo.js

```diff
@@ -44,6 +59,12 @@ export const actions = {
 };
 
 export const mutations = {
+  toggleSearchLoad(state) {
```

Comment:
`toggleSearchLoad` seems like loading of some data rather than the spinner.
```suggestion
  toggleSearchLoading(state) {
```